pontoon/batch/actions.py

```diff
@@ -135,6 +149,14 @@ def reject_translations(form, user, translations, locale):
     ]
     ActionLog.objects.bulk_create(actions_to_log)
 
+    # Send Review Master Badge notification information
+    after_level = user.badges_review_level
+    badge_update = {}
+    if after_level > before_level:
+        badge_update["level"] = after_level
+        badge_update["name"] = "Review Master Badge"
```

Comment:
Not sure why this works in other actions, but the `BadgeTooltip` component expects badge names without the word Badge at the end in `switch` - `case`.

Reply:
In `useSendTranslation` we seem to hardcode the `badgeName` rather than using the one returned from the backend?

Reply:
Great catch, I was so confused 🥲

Reply:
Let's use the same message style everywhere, i.e. drop "Badge" at the end of the badge name.

Reply:
That way we can also use the `badgeName` in `useSendTranslation`.